enclave.go

```diff
@@ -128,6 +128,12 @@ type Config struct {
 	// if the enclave application exposes an HTTP server.  Non-HTTP enclave
 	// applications can ignore this.
 	AppWebSrv *url.URL
+
+	// WaitForApp instructs nitriding to wait for the application's signal
+	// before launching the Internet-facing Web server.  Set this flag if your
+	// application takes a while to bootstrap and you don't want to risk
+	// inconsistent state when syncing, or unexpected attestation documents.
```

Comment:
Maybe mention this means the app needs to call the `/enclave/ready` endpoint to bring up the external interface?

Reply:
Fixed in d69b484.